### DISCUSSION TITLE
Facebook integration

### DISCUSSION THREAD

**Discussion by jorgeorejas** [supabase/supabase#2828] [Questions] [answered]
I would like to know if it's possible to use the Facebook OAuth to validate the Instagram or Facebook Graph API. It would be awesome to use it rather than the official one. 
[This video](https://www.youtube.com/watch?v=cMv6cCi-x-E) shows what I refer to. 

Thanks you in advance :)

**Comment by inian** [comment, marked as answer]
Hi @jorgeorejas, yeah this should be possible. You can pass in the scopes mentioned [here](https://github.com/seanlon/full-flow-facebook-instagram-graph-api/blob/7846842aec7b728742357f30878769e9890f46d8/scripts/flow-facebook-graph-instagram.js#L142) in Supabase Auth [docs](https://supabase.io/docs/reference/javascript/auth-signin#sign-in-with-scopes). 

Once you get the final access token, you should be able to make further calls to the graph and instagram API as documented in that [example repo](https://github.com/seanlon/full-flow-facebook-instagram-graph-api/blob/master/scripts/flow-facebook-graph-instagram.js)
